### PR TITLE
feat: feedback mail

### PR DESCRIPTION
**Pull request by zhouhanseng**

### CODE REVIEW

File: src/extension/options-page/DashboardDialogs/Feedback.tsx [hunk 1]
```diff
@@ -45,8 +47,10 @@ export function DashboardFeedbackDialog(props: WrappedDialogProps) {
                 footer={
                     <DebounceButton
                         variant="contained"
-                        disabled={Boolean(!name || !email || !message)}
-                        onClick={async () => {}}>
+                        disabled={Boolean(!name || !message)}
+                        onClick={async () => {
+                            window.open(`mailto:info@dimension.im?body=${message}&subject=${name}`)
```
Comment (septs) [inline src/extension/options-page/DashboardDialogs/Feedback.tsx:52]: use URL builder this string.

**Comment by septs** [comment]
plz clear warnning.

File: src/extension/options-page/DashboardDialogs/Feedback.tsx [hunk 2]
```diff
@@ -25,13 +25,15 @@ export function DashboardFeedbackDialog(props: WrappedDialogProps) {
                             value={name}
                             onChange={(e) => setName(e.target.value)}
                         />
-                        <TextField
+                        {/* Todo: fix the whole feedback loop asap, now just mailto info@dimension.im */}
+                        {/* <TextField
```
Comment (septs) [inline src/extension/options-page/DashboardDialogs/Feedback.tsx:29]:
```suggestion
                        {/* <TextField
```

why not use `hidden` attribute

File: src/extension/options-page/DashboardDialogs/Feedback.tsx (hunk 1 above)
Comment (zhouhanseng) [reply src/extension/options-page/DashboardDialogs/Feedback.tsx:52]: `new URL()` convert string to object, not the other way around.

Comment (septs) [reply src/extension/options-page/DashboardDialogs/Feedback.tsx:52]:
```
const url = new URL("mailto:info@dimension.im")
url.searchParams.set("subject", "subject")
url.searchParams.set("body", "💊")
url.toString()
// returns "mailto:info@dimension.im?subject=subject&body=%F0%9F%92%8A"
```

join strings, you have not url encode

File: src/extension/options-page/DashboardDialogs/Feedback.tsx [hunk 3]
```diff
@@ -25,7 +25,9 @@ export function DashboardFeedbackDialog(props: WrappedDialogProps) {
                             value={name}
                             onChange={(e) => setName(e.target.value)}
                         />
+                        {/* Todo: fix the whole feedback loop asap, now just mailto info@dimension.im */}
                         <TextField
+                            style={{ display: 'none' }}
```
Comment (zhouhanseng) [inline src/extension/options-page/DashboardDialogs/Feedback.tsx:30]: `hiden` attribute not work.

File: src/extension/options-page/DashboardDialogs/Feedback.tsx [hunk 4]
```diff
@@ -45,8 +47,13 @@ export function DashboardFeedbackDialog(props: WrappedDialogProps) {
                 footer={
                     <DebounceButton
                         variant="contained"
-                        disabled={Boolean(!name || !email || !message)}
-                        onClick={async () => {}}>
+                        disabled={Boolean(!name || !message)}
+                        onClick={async () => {
+                            const url = new URL('mailto:info@dimension.im')
```
Comment (septs) [inline src/extension/options-page/DashboardDialogs/Feedback.tsx:52]: https://github.com/DimensionDev/Maskbook/blob/master/src/_locales/en/messages.json#L68

File: src/extension/options-page/DashboardDialogs/Feedback.tsx (hunk 3 above)
Comment (septs) [reply src/extension/options-page/DashboardDialogs/Feedback.tsx:30]: ok

File: src/extension/options-page/DashboardDialogs/Feedback.tsx (hunk 4 above)
Comment (zhouhanseng) [reply src/extension/options-page/DashboardDialogs/Feedback.tsx]: done

**Review by septs — APPROVED** [review]

**Review by Jack-Works — APPROVED** [review]

**Pull request merged**